lib/tutorials/en_US/routing.md

```diff
@@ -183,21 +228,50 @@ The second property is `validate`.  This allows you to set validate rules for va
 
 ## <a name="missing" /> 404 Handling
 
-404 errors will happen whenever your server can't find what was a resource that was requested.  It is best practice to handle these errors the proper way.  This is easy to do in hapi, by just employing a route that will catch everything your other routes will not.  
+404 errors will happen whenever your server can't find what was a resource that was requested.  It is best practice to handle these errors the proper way.  This is easy to do in hapi, by just employing a route that will catch everything your other routes will not.  The following example shows how to use the `vision` plugin with `handlebars` to return a custom 404 page:
 
 ```js
+'use strict';
+
+const Hapi = require('hapi');
+
+const internals = {};
+
+const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+});
+
 server.route({
     method: '*',
     path: '/{any*}',
-    handler: (request, h) => {
+    handler: function (request, h) {
 
         return h.view('404').code(404);
     }
 });
+
+const init = async () => {
+
+    await server.register(require('vision'));
+
+    server.views({
+        engines: {
+            html: require('handlebars')
+        },
+        path: '../views'
+    });
+
+    await server.start();
+    console.log('Server running on %ss', server.info.uri);
+};
```

Comment:
Odd order of things. Why separate the server creation from the register? Also route using view should come after views manager initialized.

Reply:
@jarrodyellets do you plan on updating your PR according to this feedback?

Reply:
This should be updated in commit 8040e4e

Reply:
Yes indeed for the route declaration but I think what Eran also meant is that the server creation:
```js
const server = Hapi.server({
    port: 3000,
    host: 'localhost'
});
```
should also be inside the `init` method.

Reply:
Ahhh...ok, gotcha...fixing now

Reply:
Also reading again my comment, it feels a bit harsh. Hope you didn't take it the wrong way it was definitely not intended if that's the case.

Reply:
Lol, no worries...all good!

Reply:
Cool. 😃